cmd/nginx/flags.go

```diff
@@ -17,13 +17,13 @@ limitations under the License.
 package main
 
 import (
+	"encoding/json"
 	"flag"
 	"fmt"
+	"github.com/spf13/pflag"
```

Comment:
It would be better to keep 3rd party modules after the standard ones.

Reply:
Thank you. Fixed